docs/guides/universal-receiver-delegate/set-default-implementation.md

```diff
@@ -282,7 +285,7 @@ Generate _Data Keys & Values_ for [**adding a URD**](../../standards/generic-sta
   <TabItem value="web3js" label="web3.js">
 
 ```typescript title="Encode Data Keys & Values for updating the URD and its permissions"
-const addressPermissionsOldArrayLengthHex = await myUP.methods[
+const addressPermissionsOldArrayLengthHex = await universalProfile.methods[
```

Comment:
I thik with the removal of function overloading, we can just use the function name, not the function signature anymore?
Just noticed that in the review.